src/test/java/junit/samples/ListTest.java

```diff
@@ -11,8 +11,9 @@
  * A sample test case, testing {@link java.util.ArrayList}.
  */
 public class ListTest extends TestCase {
-    protected List<Integer> fEmpty;
-    protected List<Integer> fFull;
+    private List<Integer> fEmpty;
```

Comment:
While you are changing this, could you please rename "fEmpty" to "empty" or "emptyList"?